main.cpp

```diff
@@ -201,6 +201,51 @@ Thing 1) Coffee maker
     3) beep when pouring complete
  */
 
+struct CoffeeMaker
+{
+    //maximum water temperature in F
+    float maxWaterTemp = 75;
+    //cup size in mls
+    int cupSize = 200;
+    //indicate whether cup full
+    bool cupFull = false;
```

Comment:
These properties are not relevant to a coffee maker.
`Water Capacity` is relevant as is `currently brewing or not`
- [ ] revise

Reply:
Tried to come up with two other original properties.